components/sysutils/fastfetch/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,60 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Carter Li
```

Comment:
Please do not edit this file.  It should be added to git verbatim.  Otherwise `env - gmake sample-manifest` will overwrite it anyway and so you'll not pass the [basic component consistency check](https://github.com/OpenIndiana/oi-userland/pull/18013#issuecomment-2205739703) :-).

Reply:
Should I delete it?

Reply:
Simply run:
```
env - gmake clobber
env - gmake sample-manifest
git add manifests/sample-manifest.p5m
```